### ISSUE TITLE
How to fix the default settings of the model?

### ISSUE THREAD

**Issue by wszgrcy** [ollama/ollama#6169]
### What is the issue?

I found that the template for 'yi: 9b-v1.5-q8-0' is missing and different from the 'yi: 9b' template
Where should the fix be carried out?

![image](https://github.com/user-attachments/assets/90bd2fc8-b4c7-40b2-92c9-fbf9c9a08298)
![image](https://github.com/user-attachments/assets/5cec12cd-8f25-45be-8990-3eabda9c95d8)
![image](https://github.com/user-attachments/assets/1b6eeec8-dc74-4dff-8df5-bedb6ce85fde)

### OS

Windows

### GPU

AMD

### CPU

AMD

### Ollama version

0.3.2

**Comment by igorschlum** [comment]
@wszgrcy good point. I checked different tags for yi models on ollama.com and params are not consistent. All quantizations have missing params. Even the Q4_0 witch should be the same as the default one.

**Comment by wszgrcy** [comment]
> @wszgrcy good point. I checked different tags for yi models on ollama.com and params are not consistent. All quantizations have missing params. Even the Q4_0 witch should be the same as the default one.

Thank you, I see that the configuration seems to have been fixed

**Comment by igorschlum** [comment]
Are you sur? Here the params are still missing:

https://ollama.com/library/yi:9b-v1.5-q8_0

**Comment by wszgrcy** [comment]
> Are you sur? Here the params are still missing:
> 
> https://ollama.com/library/yi:9b-v1.5-q8_0

I looked at the tag I pulled and found that it was`9b-chat-v1.5-q8-0`, but`9b-v1.5-q8-0`is still missing
It seems like I pulled the wrong tag
What are the differences between these two versions

**Comment by mxyng** [comment]
This seems like a mistake in the tagging. As the name suggests `9b-chat-v1.5-q8-0` is the chat finetune of Yi 9B v1.5 while `9b-v1.5-q8-0` is the pretrained model. Previously we tagged the latter as `text`, e.g. 9b-text-v1.5-q8_0` to avoid confusion but missed it in this case

**Comment by chenhz01** [comment]
This class of problem usually comes from quantized variants shipping without the template the base model has, so ollama silently falls back to something wrong — which matches the chat-vs-pretrained tagging mistake mxyng identified. A workable stopgap while the library entry gets fixed: author an explicit Modelfile with TEMPLATE and PARAMETER pinned for the variant instead of relying on defaults, then diff a generation against the base model to confirm the fix. Could you paste the template diff you see between yi:9b and the q8_0 variant? If the chat template markers were dropped, the pinned Modelfile should restore correct behavior immediately.
